build/dependencies.props

```diff
@@ -1,9 +1,12 @@
 ﻿<Project>
   <PropertyGroup>
     <AspNetCoreVersion>1.2.0-*</AspNetCoreVersion>
```

Comment:
I'll see how much this is used now. If it's just for the `CommandLineUtils`, I'll leave it as-is. Due to patch releases, might have to use too many versions for NuGet use to be interesting and controllable from this file.

BTW I didn't change this property or add a source to the NuGet.config file 😺